GlideRecord/List of Child Incidents/listofchildIncident.js

```diff
@@ -0,0 +1,11 @@
+var parentID = "9c573169c611228700193229fff72400"; // This should be stored in sys_property instead of hardcoding the sys_id. 
+var inc = new GlideRecord('incident');
+inc.addEncodedQuery('parent_incident='+parentID); // sys_id of parent Incident
+inc.query();
+
+gs.info('List of Child Incidents of INC0000001:');
```

Comment:
This line makes no sense with this INC number. It should print the parent number dynamically. Something like this perhaps (added after the query statement):
```
if(inc.next()) {
  gs.info('List of child incidents of ' + inc.getDisplayValue('parent_incident'));
}
...
```